odfuzz/entities.py

```diff
@@ -109,15 +108,15 @@ def _append_corresponding_queryable(self, query_group):
 
 class DirectBuilder:
     """A class for building and initializing all queryable entities with metadata passed in constructor."""
-    def __init__(self, metadata, restrictions):
+    def __init__(self, metadata, restrictions,method):
```

Comment:
It would be good to check inputs here.. since it grabs "any string" - have a set of possible valid arguments, and in case somebody passes anything else then "GET" and "DELETE" at the moment, throw InvalidArgumentException.. so there is no need to debug deep into the fuzzer.py.